src/CFSsh/CFSsh.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
This needs to be a .NET 6 app, net5 is EOL

Reply:
Ah ok, we'll need to do some finagling in that case; VS 2019 doesn't officially support .net 6:

> "Targeting net6.0 is officially supported in Visual Studio 17.0+ only." 
> https://docs.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs

I made this change in the first place because the VS 2019 build failed (see pipeline failure [here](https://dev.azure.com/TanzuDevX/DevX/_build/results?buildId=840&view=logs&j=69944193-f126-526b-b021-abb6fb4b38ee&t=22fc0975-bb68-5b6e-eb1e-6f8137285546&l=35))

Reply:
see #358 and #359

Reply:
Ultimate decision (after conversation this morning) is to use .NET Framework (4.7) for now. 

Implemented in [8d51253](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/360/commits/8d5125315886a3bbfec1d868869a3bde072cefa9)